test/starlark_tests/verifier_scripts/codesignopts_verifier.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Assert that the bundle itself is signed.
+assert_is_codesigned "$BUNDLE_ROOT"
```

Comment:
this file appears to be unused?

Reply:
your pr comments are on outdated commits